modules/rostests/apitests/crt/crtdata.c

```diff
@@ -0,0 +1,640 @@
+    _CRTIMP unsigned int* ___unguarded_readlc_active_add_func(void);
+    ok_ptr(___unguarded_readlc_active_add_func(), p);
+}
+
+void Test___wargv()
+{
+    void* p = &__wargv;
+    test_is_local_symbol(p, FALSE);
+
+    #undef __wargv
+    _CRTIMP extern wchar_t ** __wargv;
+    ok_ptr(&__wargv, p);
+#ifdef _M_IX86
+    ok_ptr(__p___wargv(), p);
+#endif
+}
+
+void Test___winitenv()
+{
+    _CRTIMP extern wchar_t** __winitenv;
+    ok(__winitenv == NULL, "__winitenv is not NULL\n");
+#ifdef _M_IX86
+    _CRTIMP wchar_t*** __p___winitenv(void);
+    ok_ptr(__p___winitenv(), &__winitenv);
+#endif
+}
+
+void Test__acmdln()
+{
+    _CRTIMP extern char* _acmdln;
+    ok(_acmdln != NULL, "__winitenv is NULL\n");
+#ifdef _M_IX86
+    _CRTIMP char** __p__acmdln(void);
+    ok_ptr(__p__acmdln(), &_acmdln);
+#endif
+}
+
+#ifdef _M_IX86
+void Test__adjust_fdiv()
+{
+    _CRTIMP extern int _adjust_fdiv;
+    ok_int(_adjust_fdiv, 0);
+}
+#endif
+
+void Test__aexit_rtn()
+{
+    typedef void (*_exit_t)(int exitcode);
+    _CRTIMP extern _exit_t _aexit_rtn;
+    ok_ptr(_aexit_rtn, _exit);
+}
+
+void Test__commode()
+{
+    void* p = &_commode;
+    test_is_local_symbol(&_commode, FALSE);
+    ok_int(_commode, 0);
+
+    #undef _commode
+    _CRTIMP extern int _commode;
+    ok_ptr(&_commode, p);
+#ifdef _M_IX86
+    ok_ptr(__p__commode(), &_commode);
+#endif
+}
+
+void Test__ctype()
+{
+    _CRTIMP extern const unsigned short _ctype[];
+    ok_int(_ctype[0], 0);
+    ok_int(_ctype[1], _CONTROL);
+
+    #undef _pctype
+    _CRTIMP extern const unsigned short* _pctype;
+    ok(_pctype != &_ctype[0], "_pwctype should not match &_wctype[0]");
+    if (s_osvi.dwMajorVersion >= 6)
+    {
+        ok(_pctype != &_ctype[1], "_pwctype should not match &_wctype[1]");
+    }
+    else
+    {
+        ok(_pctype == &_ctype[1], "_pwctype should match &_wctype[1]");
+    }
+
+    ok(__pctype_func() != _ctype, "__pctype_func() should not match _ctype\n");
+    ok_int(__pctype_func()[0], _CONTROL);
+    ok_int(__pctype_func()[1], _CONTROL);
+#ifdef _M_IX86
+    _CRTIMP const unsigned short** __cdecl __p__pctype(void);
+    ok_ptr(*__p__pctype(), __pctype_func());
+#endif
+}
+
+void Test__wctype()
+{
+    ok_int(_wctype[0], 0);
+    ok_int(_wctype[1], _CONTROL);
+
+    #undef _pwctype
+    _CRTIMP extern const unsigned short* _pwctype;
+    ok_ptr(_pwctype, &_wctype[1]);
+
+    ok(__pwctype_func() != _wctype, "__pwctype_func() == _wctype\n");
+    ok_int(__pctype_func()[0], _CONTROL);
+    ok_int(__pctype_func()[1], _CONTROL);
+#ifdef _M_IX86
+    _CRTIMP const unsigned short** __cdecl __p__pwctype(void);
+    ok_ptr(*__p__pwctype(), __pwctype_func());
+#endif
+}
+
+void Test__daylight()
+{
+    void* p = &_daylight;
+    test_is_local_symbol(&_daylight, FALSE);
+
+    #undef _daylight
+    _CRTIMP extern int _daylight;
+    ok_ptr(&_daylight, p);
+
+#ifdef _M_IX86
+    _CRTIMP void* __p__daylight(void);
+    ok_ptr(__p__daylight(), &_daylight);
+#endif
+#if (WINVER >= 0x600)
```

Comment:
Above you used `WIN32_WINNT` but here and below you use `WINVER`; any reason for that?

Reply:
No.